packages.json

```diff
@@ -1900,6 +1900,7 @@
   "https://github.com/Mobelux/ImageFetcher.git",
   "https://github.com/mobelux/lingo.git",
   "https://github.com/mobileforming/WiremockClient.git",
+  "https://github.com/mochidev/Bytes.git",
```

Comment:
I'm getting a 404 from that url, @dimitribouniol

Reply:
I was pretty sure it was public from the beginning, but I guess I missed that part 😅 - Should be available now if you ran give it one more go! Sorry about that...

Reply:
Ah, I see - I can access it now. Will merge once CI completes. Thanks, Dimitri!